.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] You are doing a good job so far, 👍 👍 , As suggested by the previous reviewer, It is part of the project requirement for you to include the `test.md` in your `.gitignore` file. 😄 
> A test.md file with the text "This file should be ignored by git".
- Please consider adding `test.md` in your `.gitignore` file blow the `node_modules/` line 😄 💯